src/content/quickstart/kind/_index.md

```diff
@@ -64,6 +64,22 @@ kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 expose deployment n
 subctl export service --kubeconfig output/kubeconfigs/kind-config-cluster3 --namespace default nginx
 ```
 
+To verify a headless service, follow the steps below using a `mehdb` service
```

Comment:
Use headless nginx instead. Doesn't make sense to change the service itself to demonstrate headless use case.

Reply:
Then we need to have it in our repo right ? also this particular service can be extended for stateful sets too and it covers a real use case, where it will create the physical volumes too.

Reply:
You can pass it as a flag to expose command. Taken from `kubectl expose --help`
```  # Create a service for an nginx deployment, which serves on port 80 and connects to the containers on port 8000.
  kubectl expose deployment nginx --port=80 --target-port=8000

Options:
      --allow-missing-template-keys=true: If true, ignore any errors in templates when a field or map key is missing in
the template. Only applies to golang and jsonpath output formats.
      --cluster-ip='': ClusterIP to be assigned to the service. Leave empty to auto-allocate, or set to 'None' to create
a headless service.